### PR TITLE
More mandarin importers

### PR DESCRIPTION
**Pull request by reuben**
Would be interesting to see if we could get a license to the full DataTang corpus of 1505 hours: https://en.datatang.com/en/opensource

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
@reuben Do you think I should review this and add it as a feature for others?

**Comment by reuben** [comment]
Yeah, I think this is useful to have. Importers tend to be low/zero maintenance so no much harm in adding more.

File: bin/import_primewords.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    df.sort_values(by='wav_filename', inplace=True)
+
+    indices = np.arange(0, len(df))
+    np.random.seed(12345)
+    np.random.shuffle(indices)
+
+    test_indices = indices[-5000:]
+    dev_indices = indices[-10000:-5000]
+    train_indices = indices[:-10000]
+
```
Comment (kdavis-mozilla) [inline bin/import_primewords.py]: Why this train, dev, test split? The dev and test sizes are independent of the entire data set's size.

File: bin/import_freestmandarin.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    df.sort_values(by='wav_filename', inplace=True)
+
+    indices = np.arange(0, len(df))
+    np.random.seed(12345)
+    np.random.shuffle(indices)
+
+    test_indices = indices[-5000:]
+    dev_indices = indices[-10000:-5000]
+    train_indices = indices[:-10000]
+
```
Comment (kdavis-mozilla) [inline bin/import_freestmandarin.py]: Why this train, dev, test split? The dev and test sizes are independent of the entire data set's size.

File: bin/import_primewords.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    train_files = df.iloc[train_indices]
+    durations = (train_files['wav_filesize'] - 44) / 16000 / 2
+    train_files = train_files[durations <= 15.0]
```
Comment (kdavis-mozilla) [inline bin/import_primewords.py]: Why 15 seconds and not the more conventional 10 seconds?

Comment (reuben) [reply bin/import_primewords.py]: 10 seconds cuts off something like 30 hours of data in this dataset.

File: bin/import_primewords.py (hunk 1 above)
Comment (reuben) [reply bin/import_primewords.py]: It's just an arbitrary size based roughly on the size of LibriSpeech test-clean + test-other. For these smaller sets, trying to get a statistically sound sample size means you have to use half of the corpus or more as dev/test.

Comment (reuben) [reply bin/import_primewords.py]: ...or maybe I was just using seconds of data instead of number of samples when computing sample sizes! d'oh!

**Pull request merged**